docs/en_US/Compressor/Pruner.md

```diff
@@ -92,3 +92,55 @@ You can view example for more information
 
 ***
 
+## Filter Pruner
+
+In ['PRUNING FILTERS FOR EFFICIENT CONVNETS'](https://arxiv.org/abs/1608.08710), authors Hao Li, Asim Kadav, Igor Durdanovic, Hanan Samet and Hans Peter Graf.
+
+![](../../img/filter_pruner.png)
+
+> The procedure of pruning m filters from the ith convolutional layer is as follows:
+> 1. For each filter $F_{i,j}$ , calculate the sum of its absolute kernel weights $s_j = \sum_{l=1}^{n_i}\sum|K_l|$
+> 2. Sort the filters by $s_j$.
+> 3. Prune $m$ filters with the smallest sum values and their corresponding feature maps. The
+>   kernels in the next convolutional layer corresponding to the pruned feature maps are also
+>   removed.
+> 4. A new kernel matrix is created for both the $i$th and $i+1$th layers, and the remaining kernel
+>   weights are copied to the new model.
+
+### Usage
+
+PyTorch code
+
+```
+from nni.compression.torch import FilterPruner
+config_list = [{ 'sparsity': 0.8, 'op_types': ['Conv2d'] }]
+pruner = FilterPruner(config_list)
+pruner(model)
+```
+
+#### User configuration for Filter Pruner
+
+- **sparsity:** This is to specify the sparsity operations to be compressed to
```

Comment:
does the paper explain how sparsity is set?

Reply:
Yes, users can set the sparsity by sensitive analyse, but this is not the key point and highly coupled with user's model